.github/workflows/ci.yml

```
@@ -23,7 +23,7 @@ jobs:
       - name: Setup .NET SDK
         uses: actions/setup-dotnet@v4.2.0
         with:
-          dotnet-version: 8.0.x
+          dotnet-version: 9.0.x
```

```suggestion
          dotnet-version: |
            9.0.x
            8.0.x
```
Shouldn't we target both the net 8 and 9 for ci workflows?

---

Found out that this is not required as other projects are not targeting 8 or 9